docs/quickstart/fuji-workflow.md

```diff
@@ -35,171 +35,214 @@ const strength: number = 256
+// "chimney asset heavy ecology accuse window gold weekend annual oil emerge alley retreat rabbit seed advance define off amused board quick wealth peasant disorder"
 ```
 
 ## Derive Addresses
 
 After generating a mnemonic we can use AvalancheJS to derive [BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)-compliant hierarchical deterministic (HD) keypairs.
 
 ```typescript
-import HDNode from "avalanche/utils/hdnode"
+import HDNode from "avalanche/dist/utils/hdnode"
+import { Avalanche, Mnemonic, Buffer } from "avalanche"
+import { EVMAPI, KeyChain } from "avalanche/dist/apis/evm"
+import { ethers } from "ethers"
+
+const ip: string = "api.avax-test.network"
+const port: number = 443
+const protocol: string = "https"
+const networkID: number = 5
+const avalanche: Avalanche = new Avalanche(ip, port, protocol, networkID)
+const cchain: EVMAPI = avalanche.CChain()
+
+const mnemonic: Mnemonic = Mnemonic.getInstance()
+const m: string =
+  "chimney asset heavy ecology accuse window gold weekend annual oil emerge alley retreat rabbit seed advance define off amused board quick wealth peasant disorder"
 const seed: Buffer = mnemonic.mnemonicToSeedSync(m)
 const hdnode: HDNode = new HDNode(seed)
 
+const keyChain: KeyChain = cchain.newKeyChain()
+
+const cAddresses: string[] = []
+
 for (let i: number = 0; i <= 2; i++) {
-  // Deriving the _i_th external BIP44 X-Chain address
-  const child: HDNode = hdnode.derive(`m/44'/9000'/0'/0/${i}`)
-  keychain.importKey(child.privateKeyCB58)
+  const child: HDNode = hdnode.derive(`m/44'/60'/0'/0/${i}`)
+  keyChain.importKey(child.privateKey)
+  const cchainAddress = ethers.utils.computeAddress(child.privateKey)
+  cAddresses.push(cchainAddress)
 }
-
-const xAddressStrings: string[] = xchain.keyChain().getAddressStrings()
-console.log(xAddressStrings)
+console.log(cAddresses)
 // [
-//   'X-fuji1cfvdpdqyzpp8pq0g6trmjsrn9pt8nutsfm7a40',
-//   'X-fuji1y75dj6qygj7frw2xtcfn724qfty4aadnmeth6y',
-//   'X-fuji1p6n0vyjqgmp06f7pr405q2flqu9v93j383ncam'
+//   '0x2d1d87fF3Ea2ba6E0576bCA4310fC057972F2559',
+//   '0x25d83F090D842c1b4645c1EFA46B15093d4CaC7C',
+//   '0xa14dFb7d8593c44a47A07298eCEA774557036ff3'
 // ]
 ```
 
-**Note** that we're using `keychain` which hasn't been defined yet. Creating an empty keychain can be seen in [this example AvalancheJS script](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/newKeyChain.ts). There are links to dozens of AvalancheJS examples in [the resources listed below](fuji-workflow.md#resources).
+**Note** that we're using `keyChain` which has been defined as a new instance of a `C Chain KeyChain`. Another example of creating an empty keychain can be seen in [this example AvalancheJS script](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/newKeyChain.ts). There are links to dozens of AvalancheJS examples in [the resources listed below](fuji-workflow.md#resources).
```

Comment:
https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/newKeyChain.ts is for X-Chain (AVM). As we are changing to C-Chain, do we have C-Chain examples? If not, we can remove this sentence.